doc/_static/zotero.bib

```diff
@@ -1035,6 +1035,15 @@ @misc{g4prm
   collaboration = {Geant4}
 }
 
+@misc{g4vg-1-0,
+  title = {{{G4VG}} 1.0},
+  author = {Seth R. Johnson and Guilherme Lima and Ben Morgan and Johnson, Seth R. and Morgan, Ben},
```

Comment:
```suggestion
  author = {Seth R. Johnson and Guilherme Lima and Ben Morgan},
```

Reply:
Don't I get double credit?!

Reply:
For serious though this has to do with the zotero -> DOI -> zenodo -> TeX pipeline: https://zenodo.org/records/15450422 lists me twice (once for authoring, once for reviewing). The Celeritas records show both of us twice, as well. I guess maybe 'editor' isn't the right role? 😕 I think it's a bug in Zotero... I'll see if I can fix it.

Reply:
That took way too long: https://github.com/celeritas-project/celeritas/commit/308996c1b

Reply:
Heh well it's good to do, it would've been embarrassing to have it show up in a publication 😅